app/uk/gov/hmrc/nationalinsurancerecord/controllers/DocumentationController.scala

```diff
@@ -16,13 +16,15 @@
 
 package uk.gov.hmrc.nationalinsurancerecord.controllers
 
-import play.api.http.{HttpErrorHandler, LazyHttpErrorHandler}
+import com.google.inject.{Inject, Singleton}
+import play.api.http.HttpErrorHandler
 import play.api.mvc.{Action, AnyContent}
 import uk.gov.hmrc.nationalinsurancerecord.config.{APIAccessConfig, AppContext}
 import uk.gov.hmrc.nationalinsurancerecord.domain.APIAccess
 import uk.gov.hmrc.nationalinsurancerecord.views._
 
-class DocumentationController (errorHandler: HttpErrorHandler, appContext: AppContext)
+@Singleton
+class DocumentationController @Inject()(errorHandler: HttpErrorHandler, appContext: AppContext)
```

Comment:
Needs module to bind LazyHttpErrorHandler

Reply:
As discussed this is already bound elsewhere